Lavinia-api/Models/DistrictMetrics.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LaviniaApi.Models
+{
+    // API v2
+    public class DistrictMetrics
```

Comment:
A lot of these models are missing comments that describe what the fields mean, and what they're measured in. Integers and strings aren't very descriptive (ideally one might suggest using classes to encapsulate these fields and use utilities for validation, but that is for another time).